lib/plugins/slack/attachements.js

```diff
@@ -93,10 +93,20 @@ module.exports = {
       let color = 'danger';
 
       const limitMetric = metrics[options.limitMetric];
-      if (limitMetric) {
+      if (limitMetric && options.limitMetric === 'coach') {
```

Comment:
This check would be:
<= error: 'danger'
else if
<= warning: 'warning'
(no else needed if 'good' is default)